extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -133,6 +146,14 @@ export class AmpA4A extends AMP.BaseElement {
     /** @private {!Array<!Promise<!Array<!Promise<?PublicKeyInfoDef>>>>} */
     this.keyInfoSetPromises_ = this.getKeyInfoSets_();
 
+    // TODO(tdrl): Temporary, while we're verifying whether this is an
+    // acceptable solution to the 'Safari on iOS doesn't fetch iframe src
+    // from cache' issue.  See https://github.com/ampproject/amphtml/issues/5614
+    /** @private {?ArrayBuffer} */
+    this.creativeBody_ = null;
+    /** @private {?string} */
+    this.shouldRenderViaSafeFrame_ = null;
```

Comment:
Either this should be a boolean or renamed (e.g. experimentalNonAmpCreativeRenderMethod_).